operator/controllers/seldondeployment_controller.go

```diff
@@ -679,6 +680,30 @@ func createContainerService(deploy *appsv1.Deployment, p machinelearningv1.Predi
 			SessionAffinity: corev1.ServiceAffinityNone,
 		},
 	}
+	switch *pu.Type {
+	case machinelearningv1.ROUTER:
```

Comment:
Can we move to a separate file with some tests?

Reply:
@cliveseldon Fixed